### PR TITLE
Minor editorial cleanup

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
@datacarpentry/owners - Since I'm the maintainer of this material and we have a don't merge your own PRs rule I'd appreciate reviews and/or merges of this and other PRs I'm submitting to this repo. These are all based on teaching this material last week.

File: sql.md [hunk 1]
```diff
@@ -26,23 +26,24 @@ Relational databases
...
---------
+Dataset Description
+-------------------
 
-This is data on a small mammal community in southern Arizona over the last 35
-years.  This is part of a larger project studying the effects of rodents and
-ants on the plant community.  The rodents are sampled on a series of 24 plots,
-with different experimental manipulations of which rodents are allowed to access
-the plots.
+The data we will be using is time-series for a a small mammal community in
```
Comment (hlapp) [inline sql.md]: is _a_ time-series?

File: sql.md [hunk 2]
```diff
@@ -26,23 +26,24 @@ Relational databases
...
+southern Arizona. This part of a project studying the effects of rodents and
```
Comment (hlapp) [inline sql.md]: This _is_ part of a

File: sql.md [hunk 3]
```diff
@@ -291,9 +291,8 @@ using a GROUP BY clause
 GROUP BY tells SQL what field or fields we want to use to aggregate the data.
 If we want to group by multiple fields, we give GROUP BY a comma separated list.
 
-***EXERCISE: Write queries that return:***
-***1. How many individuals were counted in each year***
-***2. Average weight of each species in each year**
+***EXERCISE: Write queries that return: 1. How many individuals were counted in
+   each year. 2. Average weight of each species in each year**
```
Comment (hlapp) [inline sql.md]: Why does this have `***` at the front and `**` at the tail end?

**Comment by hlapp** [comment]
Looks all good aside from the minor comments I made. :+1:

**Comment by ethanwhite** [comment]
Thanks @hlapp. I just fixed the old typos you noted and one additional one that I noticed in the processs.

**Pull request merged**